pkg/corerp/frontend/controller/secretstores/types.go

```diff
@@ -19,4 +19,11 @@ package secretstores
 const (
 	// ResourceTypeName is the resource type name for secret stores.
 	ResourceTypeName = "Applications.Core/secretStores"
+
+	// The following are possible required keys in a SecretStore depending on it's SecretType
+	RequiredUsername = "username"
+	RequiredPassword = "password"
+	RequiredClientId = "clientId"
+	RequiredTenantId = "tenantId"
+	RequiredRoleARN  = "roleARN"
```

Comment:
The prefix `Required` is confusing, maybe I'm not following the thought process? They are the expected keys in the resource, so in general for such variables the convention is to add a `key` suffix, `<key_name>Key`, so it would be `UsernameKey` for example.

Reply:
updated in in https://github.com/radius-project/radius/pull/7867